src/test/resources/TestNG.xml

```diff
@@ -7,8 +7,8 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
-            <class name="Hmwk15"></class>
-            <class name="hmwk16"></class>
+            <class name="Hmwk17"></class>
+
```

Comment:
There is an error:

![image](https://user-images.githubusercontent.com/104761482/199167966-41ea4fa2-b09a-412d-aa6e-725b2e4a5c82.png)

Please add  ` <parameter name="baseURL" value="https://bbb.testpro.io/"/>`

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <parameter name="baseURL" value="https://bbb.testpro.io/"/>
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="LoginTests"/>
            <class name="BaseTest"/>
            <class name="Hmwk17"></class>
        </classes>
    </test>
</suite>
```